account_move_mail_template_with_subject/models/account_move.py

```diff
@@ -0,0 +1,17 @@
+from odoo import fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    def _get_mail_template(self):
+        """
+        :return: the correct mail template based on the current move type
+        """
+        return (
+            "account_move_mail_template_with_subject.email_template_credit_note"
+            if all(move.move_type == "out_refund" for move in self)
+            else "account_move_mail_template_with_subject.email_template_send_invoice"
+        )
```

Comment:
Avoid complete override as a general guideline.

@AungKoKoLin1997 Please help explain how this should be handled.

Reply:
Im sorry for that, i was thinking to mix those module into one module since the account_move_subject and account_move_mail_template_with_subject using same models, account_move.

Please let me confirm, So in this case i need to split into two module as the original one ( account_move_subject and account_move_mail_template_with_subject) and also for the sale order module need to split same with the account_move. and after that add those 4 modules into fal-custom as follow:
1. account_move_subject
2. account_move_mail_template_with_subject
3. sale_order_subject
4. sale_order_mail_template_with_subject

Thank you, please correct me if im wrong.